pages/common/hostapd.md

```diff
@@ -0,0 +1,11 @@
+# hostapd
+
+> Create an access point out of wireless interface.
```

Comment:
Perhaps this will read better?
```suggestion
> Start an access point using a wireless interface.
> More information: <https://w1.fi/hostapd/>.
```

Reply:
Hi. Yes, that is better.